helm/cluster-openstack/templates/kubeadm_control_plane.yaml

```diff
@@ -42,6 +42,8 @@ spec:
           oidc-client-id: {{ .clientId }}
           oidc-username-claim: {{ .usernameClaim }}
           oidc-groups-claim: {{ .groupsClaim }}
+          {{- if .usernamePrefix }}
+          oidc-username-prefix: {{ .usernamePrefix }}
```

Comment:
There is a missing `{{- end }}` here.

Reply:
Yes, missed that one, but isn't there an extra `{{- end }}` on lines 50-52 now? There are 3 `if`s and 4 `end`s in that block now.

Reply:
@erkanerol I noticed the PR is now merged, though still think one of the `{{- end }}`s from lines 50-52 has to be removed, right?